pyrdp/mitm/cli.py

```diff
@@ -127,6 +127,8 @@ def buildArgParser():
         "--transparent", help="Spoof source IP for connections to the server (See README)", action="store_true")
     parser.add_argument("--no-gdi", help="Disable accelerated graphics pipeline (MS-RDPEGDI) extension",
                         action="store_true")
+    parser.add_argument("--redirection-host", help="Redirection target ip if NLA is enforced", default=None)
+    parser.add_argument("--redirection-port", help="Redirection target port if NLA is enforced", type=int, default=None)
```

Comment:
I think we should name the flags `--nla-redirection-host` and `--nla-redirection-port` to make explicit that it is used when NLA nego fails.